src/saturn_engine/worker_manager/api/jobs.py

```diff
@@ -93,3 +100,35 @@ def post_fetch_states() -> Json[FetchCursorsStatesResponse]:
             session=session,
         )
     return jsonify(FetchCursorsStatesResponse(cursors=cursors))
+
+
+@bp.route("/_start", methods=("POST",))
+def post_start_job() -> Json[JobsStartResponse]:
+    restart: bool = flask.request.args.get("_restart", "false") == "true"
+    start_input = marshall_request(StartJobInput)
+    if not start_input.name and not start_input.job_definition_name:
+        abort(http_code=400, error_code="MUST_SPECIFY_JOB")
+
+    with session_scope() as session:
+        job: Job | None = None
+        if start_input.name:
+            job = jobs_store.get_job(session=session, name=start_input.name)
+        elif start_input.job_definition_name:
+            job = jobs_store.get_last_job(
+                session=session, job_definition_name=start_input.job_definition_name
+            )
+
+        if not job:
```

Comment:
The job might have never been started, we need to check in static definition too. Maybe store operation that wrap everything, this function is going to get big. Maybe something like `jobs_store.start(queue_item: QueueItem, session: Session, restart: bool)` where `queue_item` is from either `status_definition.jobs[job_name]` or `status_definition.job_definitions[job_defitinion_name].template`.